monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,80 @@
+[id="configuring-the-monitoring-stack"]
+= Configuring the monitoring stack
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: configuring-the-monitoring-stack
+
+toc::[]
+
+This section explains what is supported for the monitoring of user-defined projects. It also shows how to configure the monitoring stack, and demonstrates several common configuration scenarios.
+
+[id="configuring-the-monitoring-stack-prerequisites"]
+== Prerequisites
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in link:https://docs.openshift.com/container-platform/4.7/scalability_and_performance/scaling-cluster-monitoring-operator.html[Scaling the Cluster Monitoring Operator] and verify that you have sufficient resources.
```

Comment:
From [this comment thread](https://github.com/openshift/openshift-docs/pull/30483#discussion_r596949022) my understanding is that this is not required. If this prerequisite can be removed, then you could remove the whole  `== Prerequisites` section, given that there is only one prerequisite at the moment.

Reply:
Thanks for catching that. You are correct. I removed it.